modules/automatically-scaling-machines-to-available-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="automatically-scaling-machines-to-available-bare-metal-hosts_{context}"]
+= Automatically scaling machines to the number of available bare metal hosts
+
+To automatically create the number of `Machine` objects that match the number of available `BareMetalHost` objects, add a `metal3.io/autoscale-to-hosts` annotation to the `MachineSet` object.
+
```

Comment:
about CLI testings I think we need other QE's review since we only covered UI features

Reply:
OK, who can test these CLI parts?

Reply:
/lgtm